pyproject.toml

```diff
@@ -12,7 +12,7 @@ python = ">=3.7,<3.10"
 type-infer = "^0.0.9"
 numpy = "^1"
 pandas = "^1"
-scikit-learn = ">=1.0.0, <=1.0.2"
+scikit-learn = "0.24.2"
```

Comment:
Let's change this to `>= 0.24.2`, that way you can finish the handlers that require this while enabling other users to keep the newest (and better) versions of sklearn.

Reply:
Hey @paxcema, you are absolutely right.
I just wanted to know, will changing only the `pyproject.toml` file solve this? Or will we need to make changes to the `poetry.lock` as well?
I think we might need to execute `poetry add  "scikit-learn>=0.24.2"` and allow the relevant changes to be made to the files?

Reply:
Indeed, we should update the `poetry.lock` file. I believe you can manually change the `.toml`, then do `poetry lock` from the root folder in your terminal to update the file (you could also remove it and generate it from scratch, not sure which approach is faster but it shouldn't take too long).

Reply:
@paxcema I just pushed another commit. Please take a look.